haystack/nodes/retriever/_embedding_encoder.py

```diff
@@ -211,7 +211,7 @@ def embed_documents(self, docs: List[Document]) -> np.ndarray:
         :param docs: List of documents to embed.
         :return: Embeddings, one per input document, shape: (documents, embedding_dim)
         """
-        passages = [[d.meta["name"] if d.meta and "name" in d.meta else "", d.content] for d in docs]
+        passages = [d.content for d in docs]
         return self.embed(passages)
```

Comment:
You could check whether we can also simplify the signature of the self.embed method now. It is `def embed(self, texts: Union[List[List[str]], List[str], str]` right now and there is a comment `# texts can be a list of strings or a list of [title, text]`, which we should change.